keyboards/nullbitsco/snap/keymaps/bongo_reactive/bongo_graphics.h

```diff
@@ -0,0 +1,246 @@
+#pragma once
```

Comment:
Could you add a GPL2+ compatible license header here?  All of the source files need to have a license explicitly listed.

For instance:
```c
// Copyright 2021 Your Name (@yourgithub)
// SPDX-License-Identifier: GPL-2.0-or-later
```

Reply:
No problem! I found a few other source files missing a header as well. They're all added now.